test/integration/node-specific/auto_connect.test.ts

```diff
@@ -891,12 +891,36 @@ describe('When executing an operation for the first time', () => {
         it('client.connect() takes as long as selectServer is delayed for and does not throw a timeout error', async function () {
           const start = performance.now();
           expect(client.topology).to.not.exist; // make sure not connected.
-          const res = await client.db().collection('test').insertOne({ a: 1 }, { timeoutMS: 1000 }); // auto-connect
+          const res = await client.db().collection('test').insertOne({ a: 1 }, { timeoutMS: 500 }); // auto-connect
           const end = performance.now();
           expect(res).to.have.property('acknowledged', true);
-          expect(end - start).to.be.within(2000, 2500); // timeoutMS is 1000, did not apply.
+          expect(end - start).to.be.within(1000, 1500); // timeoutMS is 1000, did not apply.
         });
       }
     );
+
+    describe('when auto connect is used and connect() takes longer than timeoutMS', function () {
+      // This test stubs the connect method to check that connect() does not get timed out
+      // vs. the test above makes sure that the `ping` does not inherit the client's timeoutMS setting
```

Comment:
Can we make this clear with the test descriptions instead of a comment?  It seems like the second sentence of this comment is the actual intent of the previous test

```typescript
describe('when auto connect is used', async function() {
  it('timeoutMS from the client is not used for the internal `ping`', ... {} );
});
```

Reply:
updated!